mongodbatlas/data_federation.go

```diff
@@ -271,3 +288,112 @@ func (s *DataFederationServiceOp) Delete(ctx context.Context, groupID, name stri
 
 	return resp, err
 }
+
+// ConfigureQueryLimit Creates or updates one query limit for one federated database instance.
+//
+// See more: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Data-Federation/operation/createOneDataFederationQueryLimit
+func (s *DataFederationServiceOp) ConfigureQueryLimit(ctx context.Context, groupID, name, limitName string, queryLimit *DataFederationQueryLimit) (*DataFederationQueryLimit, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupID", "must be set")
+	}
+	if name == "" {
+		return nil, nil, NewArgError("name", "must be set")
+	}
+	if limitName == "" {
+		return nil, nil, NewArgError("limitName", "must be set")
+	}
+	if queryLimit == nil {
+		return nil, nil, NewArgError("queryLimit", "must be set")
+	}
```

Comment:
I don't see `queryLimit` in https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Data-Federation/operation/createOneDataFederationQueryLimit. Is it missing in the doc?

Reply:
This is basically for the Request Body Schema for this API, which is the same data structure as this entity in the Open API spec. Let me know if this should be handled differently.